<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="BackGroundFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>325</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="TopFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>380</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.477273, x2:1, y2:0.471591, stop:0 rgba(80, 174, 247, 255), stop:1 rgba(15, 143, 242, 255));
border: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="LoginNameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:17px;</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CloseButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../statics/imgs/close.png</normaloff>../statics/imgs/close.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="BodyFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>380</width>
      <height>275</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="TitleLabel">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>20</y>
       <width>221</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font-size: 18px;
</string>
     </property>
     <property name="text">
      <string>国联恶意代码辅助监测系统</string>
     </property>
    </widget>
    <widget class="QLabel" name="VersionLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>54</y>
       <width>66</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(43, 128, 247);
background-color: rgb(233, 242, 254);
border-radius: 13;
font-size: 14px;</string>
     </property>
     <property name="text">
      <string>   主机版</string>
     </property>
    </widget>
    <widget class="QFrame" name="UserNameFrame">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>94</y>
       <width>301</width>
       <height>36</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 245, 245);
border: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="UserNameInput">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>261</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
color: black;</string>
      </property>
      <property name="placeholderText">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLabel" name="UserNameIcon">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>7</y>
        <width>20</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../statics/imgs/userinput.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="PasswordFrame">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>150</y>
       <width>301</width>
       <height>36</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 245, 245);
border: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="PasswordInput">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>261</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
border: none;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QLabel" name="PasswordIcon">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>7</y>
        <width>20</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../statics/imgs/passinput.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="LoginButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    font-size: 16px;
	border: none;
	border-radius: 15;
	background-color: rgb(43, 128, 247);
}
:hover{
    font-size: 16px;
	background-color: rgb(146, 212, 255);
}


</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="QuitButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    font-size: 16px;
	border: none;
	background-color: rgb(246, 41, 41);	
	border-radius: 15;
}
:hover{
    font-size: 16px;
	background-color: rgb(255, 188, 226);
}

</string>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QLabel" name="MessageLabel1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string>MessageLabel1</string>
     </property>
    </widget>
    <widget class="QLabel" name="MessageLabel2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string>MessageLabel2</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
